rockcraft.yaml

```diff
@@ -18,10 +17,9 @@ parts:
     plugin: go
     source: https://github.com/prometheus/prometheus
     source-type: git
-    source-tag: "v2.43.0"
+    source-tag: "v2.44.0"
     build-snaps:
-      - go/1.18/stable
-      - node/18/stable
```

Comment:
At first sight the diff looks incorrect. @lucabello ?

Reply:
This is correct in the sense that Prometheus v2.44.0 builds with [go 1.19](https://github.com/prometheus/prometheus/blob/v2.44.0/go.mod#L3).